pkg/storaged/client.js

```diff
@@ -438,7 +438,7 @@ function init_model(callback) {
     }
 
     function enable_pk_features() {
-        return PK.detect().then(function (available) { client.features.packagekit = available });
+        return PK.detect().then(function (available) { console.log(available); client.features.packagekit = available });
```

Comment:
Please either remove the log or add a text to it that says what the true/false value means.

Reply:
Upps.